client/src/pages/Profile/DetailsCard/DetailsCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Paper, Typography, Stack, Avatar, Grid } from '@mui/material';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+
+export default function DetailsCard(): JSX.Element {
+  const coverImg = 'https://images.unsplash.com/photo-1625603736199-775425d2890a';
+  const avatarImg = 'https://images.unsplash.com/photo-1488716656724-3c8820d714a0';
+
+  const images = [
+    'https://images.unsplash.com/photo-1610968755695-d7fcb5fd4b92',
+    'https://images.unsplash.com/photo-1587300003388-59208cc962cb',
+  ];
+  const name = 'Norma Byers';
+  const subtitle = 'Loving pet sitter';
+  const location = 'Toronto, Ontario';
+  const description = `Animals are my passion! I will look after your pets with loving care. 
+  I have some availability for pet care in my home as well. I have 10 yrs experience at the Animal Hospital, 
+  and have owned multiple pets for many years, including numerous rescues. Kindly email, text, or call me and I will respond promptly!`;
+
+  return (
+    <Paper elevation={3}>
```

Comment:
Consider using the `Card` API from MUI as this component contains information specific to a topic

Ref: https://mui.com/api/card/